api/config/runtime.exs

```diff
@@ -21,77 +21,94 @@ if System.get_env("PHX_SERVER") do
 end
 
 if config_env() == :prod do
-  database_url =
-    System.get_env("DATABASE_URL") ||
-      raise """
-      environment variable DATABASE_URL is missing.
-      For example: ecto://USER:PASS@HOST/DATABASE
-      """
+  if System.get_env("BOOTSTRAPPED") do
+    config :api, Api.Repo,
+      username: System.get_env("DB_USER"),
+      password: System.get_env("DB_PASSWORD"),
+      hostname: System.get_env("DB_HOST"),
+      database: System.get_env("DB_NAME"),
+      port: System.get_env("DB_PORT"),
+      stacktrace: true,
+      show_sensitive_data_on_connection_error: true,
+      pool_size: 10
 
-  maybe_ipv6 = if System.get_env("ECTO_IPV6") in ~w(true 1), do: [:inet6], else: []
+    config :api, ApiWeb.Endpoint,
+      http: [ip: {0, 0, 0, 0}, port: 4000],
+      cache_static_manifest: "priv/static/cache_manifest.json",
+      server: true
+  else
```

Comment:
it's a bit hard to read this: is some of the configuration the same between the two cases?

Reply:
i'm not really sure what needs to go in the runtime portion when we're on aws. i've noticed that other repos don't even have this runtime.exs file.

i thought about deleting it altogether, but didn't have a good justification for doing so.

Reply:
You definitely can't delete it (since you need to tell the application where to access the database). I took a look in my editor and this looks okay: we can clean it up later if we need to.